user_guide_src/source/incoming/message/005.php

```diff
@@ -2,5 +2,7 @@
 
 echo $message->getHeaderLine('Accept-Language');
 
-// Outputs:
-en, en-US
+/*
+    Outputs:
+    en, en-US
```

Comment:
This is a bug in the current document.
```suggestion
    'en,en-US'
```